openid-4-verifiable-presentations-1_0.md

```diff
@@ -911,11 +917,11 @@ The following is a list of profiles that define static configuration values of W
 
 ### A Set of Static Configuration Values bound to `openid4vp://` {#openid4vp-profile}
 
-The following is a non-normative example of a set of static configuration values that can be used with `vp_token` parameter as a supported Response Type, bound to a custom URL scheme `openid4vp://` as an Authorization Endpoint:
+Below is a non-normative example of a set of static configuration values that can be used with `vp_token` parameter as a supported Response Type, bound to the `openid4vp` Authorization Endpoint in the form of custom URL scheme. 
 
 ```json
 {
-  "authorization_endpoint": "openid4vp:",
+  "authorization_endpoint": "openid4vp://",
```

Comment:
we had a long debate about this and agreed that `openid4vp:` is a way to express this, not `openid4vp://`
```suggestion
  "authorization_endpoint": "openid4vp:",
```

Reply:
we don't say `https://` in the spec, we just say `https` scheme